### DISCUSSION TITLE
your Confused ? WTF

### DISCUSSION THREAD

**Discussion by TomTravis** [dnschneid/crouton#4741] [Ideas]

**Comment by TomTravis** [comment]
![Screenshot 2022-05-19 12 20 46 PM](https://user-images.githubusercontent.com/46386261/169385923-d34a1cb4-9d03-4c5c-a775-03e5ebd28714.png)
crostini systemD PID 1 has started

**Comment by TomTravis** [comment]
![Screenshot_2022-05-19_12-38-29](https://user-images.githubusercontent.com/46386261/169388972-2cb842ec-246a-47ee-aaca-6f189e4626dc.png)

[rc.log](https://github.com/dnschneid/crouton/files/8733740/rc.log)
xorg has systemd now

**Comment by dnschneid** [comment]
Yes, that's how crostini works.  This would be a more appropriate topic for crostini discussion forums (like r/crostini on reddit).
